Sources/Purchasing/OfferingsManager.swift

```diff
@@ -78,6 +73,10 @@ class OfferingsManager {
         fetchPolicy: FetchPolicy = .default,
         completion: (@MainActor @Sendable (Result<Offerings, Error>) -> Void)?
     ) {
+        Logger.debug(isAppBackgrounded
+                     ? Strings.offering.offerings_stale_updating_in_background
+                     : Strings.offering.offerings_stale_updating_in_foreground)
```

Comment:
I guess `isAppBackgrounded` could be a parameter to build the string... But this is how it was before so we can keep as is for now.